content/zh-cn/docs/reference/config-api/kubelet-config.v1.md

```diff
@@ -0,0 +1,253 @@
+<!--
+title: Kubelet Configuration (v1)
+content_type: tool-reference
+package: kubelet.config.k8s.io/v1
+auto_generated: true
+-->
+
+---
+title: Kubelet 配置 (v1)
+content_type: tool-reference
+package: kubelet.config.k8s.io/v1
+auto_generated: true
+---
```

Comment:
Line 8-13 shall be moved to the top, that is, starting from line 1.

Reply:
got it